tests/sentry/manager/tests.py

```diff
@@ -20,16 +20,25 @@ def test_add_tags(self):
         environment = self.create_environment()
 
         with self.tasks():
-            Group.objects.add_tags(group, environment, tags=(
-                ('foo', 'bar'), ('foo', 'baz'), ('biz', 'boz')))
+            Group.objects.add_tags(
+                group,
+                environment,
+                tags=[
+                    ('foo', 'bar'),
+                    ('foo', 'baz'),
+                    ('biz', 'boz'),
+                ],
+            )
 
         results = sorted(
             tagstore.get_group_tag_values(
                 group.project_id,
                 group.id,
                 environment_id=None,
-                key='foo'),
-            key=lambda x: x.id)
```

Comment:
This was weird.

Reply:
If you implemented `__lt__` and `__gt__` on TagType, you can get rid of the key lambda I guess L)